static/styles/_components/_states.scss

```diff
@@ -3,6 +3,10 @@
   display: none;
 }
 
+.visible {
```

Comment:
Theres a .hidden selector in base/_utilities. Should that and others be moved here to this states file?